From aee9e90c1ce0257d12ea85029435db8ffa65a6a1 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Thu, 23 Apr 2020 09:31:11 +0200
Subject: Use unnamespaced hex for Qt < 5.14

Qt 5.14 moves hex under a Qt namespace, so use it in the right way
depending on the Qt version.
---
 src/utils/iso5426converter.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/utils/iso5426converter.cpp b/src/utils/iso5426converter.cpp
index 7ea81c7..0c19767 100644
--- a/src/utils/iso5426converter.cpp
+++ b/src/utils/iso5426converter.cpp
@@ -1211,7 +1211,11 @@ QChar Iso5426Converter::getCombiningChar(uint c) {
     return 0x1EF1; // SMALL LETTER U WITH HORN AND DOT BELOW
 
   default:
+#if (QT_VERSION < QT_VERSION_CHECK(5, 14, 0))
+    myDebug() << "no match for" << hex << c;
+#else
     myDebug() << "no match for" << Qt::hex << c;
+#endif
     return QChar();
   }
 }
-- 
cgit v1.1

